test.js

```diff
@@ -23,32 +23,34 @@ before(function (cb) {
 	mockServer.on('stdout', process.stdout.write.bind(process.stdout));
 	mockServer.on('stderr', process.stderr.write.bind(process.stderr));
 
-	setTimeout(function () {
+	setTimeout(() => {
 		ftp = new JSFtp({
 			host: 'localhost',
 			port: 3334,
 			user: 'test',
 			pass: 'test'
 		});
 
-		cb();
+		t.end();
```

Comment:
You need that since it's a `cb` test. Use async function and normal test instead and https://github.com/sindresorhus/delay